http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -260,7 +261,9 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       // Note: the general pattern is to 404 (e.g. pretend the runtime doesn't exist) if the caller doesn't have
       // GetClusterStatus permission. We return 403 if the user can view the runtime but can't perform some other action.
 
-      listOfPermissions <- authProvider.getRuntimeActions(runtime.samResource, userInfo)
+      listOfPermissions <- authProvider.getRuntimeActionsWithProjectFallback(googleProject,
```

Comment:
Minor comment: technically we shouldn't need the project fallback for stop/start/update because there is no project-level permission for those actions. So we could avoid a Sam call for those cases. It's probably fine as is though